# Yaml file with configutration of sensored setup
#
# markerFixList: cointains the label of the
# markers that are fixed on the setup
# markerFixPos_##: contains values that are
# a geometry_msgs::Pose: - x, y, z, qx, qy, qz, qw

# for example:
# markerFixPos_23: [0.1, 0.1, 0.0, 0.0, 0.0, 0.0, 1.0]
# where the fix marker 23 is located at position
# x, y, z =>  0.1, 0.1 , 0.0 in meters
# with the rotation
# x, y, z, q => 0.0, 0.0, 0.0, 1.0

# camera_frame: contains the string that define
# the camera frame wrt the markers are referred

aruco_broadcaster_node:
  ros__parameters:
    markerList: []

    camera_frame: rs_camera_link
    ref_frame: world
    aruco_frame: aruco_frame
